app/models/schedulable_cutoff_date.rb

```diff
@@ -0,0 +1,4 @@
+# frozen_string_literal: true
+
+class SchedulableCutoffDate < ApplicationRecord
```

Comment:
```suggestion
class SchedulableCutoffDate < CaseflowRecord
```
This will allow for this class/table to be picked up by our ERD diagraming workflows.